Resources/Prototypes/Floof/Entities/Objects/Specific/Hydroponics/equiptments.yml

```
@@ -0,0 +1,23 @@
+- type: entity
+  parent: PlantBag
+  id: ClothingHarvestBag
+  name: Harvest bag
+  description: A bag design by SESWC using bluespace technology to assisted and help botanists everyday needs.
```

Grammar and uh... whatever you call it.
```suggestion
  description: A plant bag designed by SESWC that uses bluespace technology to help botanists easily move their huge harvests.
```